src/kernels/kernelAutoRestartMonitor.node.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import type { KernelMessage } from '@jupyterlab/services';
+import { inject, injectable } from 'inversify';
+import { IExtensionSyncActivationService } from '../platform/activation/types';
+import { IDisposable, IDisposableRegistry } from '../platform/common/types';
+import { DataScience } from '../platform/common/utils/localize';
+import { IStatusProvider } from '../platform/progress/types';
+import { getDisplayNameOrNameOfKernelConnection } from './helpers.node';
```

Comment:
Same as the previous, this file should work even in web, however `getDisplayNameOrNameOfKernelConnection` is only available in node.